src/System.Runtime.Extensions/src/System/AppDomainUnloadedException.cs

```diff
@@ -0,0 +1,43 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+/*=============================================================================
+**
+** 
+**
+** Purpose: Exception class for attempt to access an unloaded AppDomain
+**
+**
+=============================================================================*/
+
+namespace System {
+
+    using System.Runtime.Serialization;
```

Comment:
Nit: would be nice if the formatting was cleaned up to follow the corefx [coding style](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md).

Some examples:

> We use [Allman style](https://en.wikipedia.org/wiki/Indent_style#Allman_style) braces, where each brace begins on a new line.

> Namespace imports should be specified at the top of the file, outside of namespace declarations and should be sorted alphabetically.

> We use language keywords instead of BCL types (i.e. int, string, float instead of Int32, String, Single, etc) for both type references as well as method calls (i.e. int.Parse instead of Int32.Parse). See issue 391 for examples.